core/src/test/resources/newlines/alwaysBeforeTopLevelStatements.stat

```diff
@@ -55,9 +61,17 @@ package a {
 
       case class C1
 
+      sealed trait T1
+
+      final class FC1
```

Comment:
Yes, that's correct, since this Test just tests if the new formatter doesn't add any new lines in this case